config/main.py

```diff
@@ -793,6 +793,8 @@ def _wait_until_clear(tables, interval=0.5, timeout=30, verbose=False):
                     click.echo("Some entries matching {} still exist: {}".format(table, keys[0]))
                 time.sleep(interval)
         empty = (non_empty_table_count == 0)
+    if timeout == 0:
+        empty = True
```

Comment:
This code looks a little wired. How about checking the value of `timeout` and return `True` directly at the beginning of the function of ` _wait_until_clear`? Or just skip calling this function `_wait_until_clear` when `timeout == 0`?

Reply:
Updated, Please check

Reply:
@bingwang-ms Can we signoff this PR?